charts/aws-ebs-csi-driver/templates/metrics.yaml

```diff
@@ -38,6 +38,10 @@ spec:
     - targetPort: 3301
       path: /metrics
       interval: {{ .Values.controller.serviceMonitor.interval | default "15s"}}
+      {{- if .Values.controller.serviceMonitor.extraRelabelings }}
```

Comment:
Sorry for the late review - rather than `extraRelabelings`, can we just call this `relabelings`? As there aren't any existing. Otherwise lgtm

Reply:
yes, sure, makes sense for me